charts/sftpgo/Chart.yaml

```diff
@@ -19,10 +19,10 @@ maintainers:
     url: https://sagikazarmark.hu
 annotations:
   artifacthub.io/changes: |
-    - kind: added
-      description: Config for deployment strategy and init containers
+    - kind: fixed
+      description: Match the app.kubernetes.io/version label to the image tag
```

Comment:
Would you mind adding the updated app version to the change log? Thanks!

Reply:
Pushed up the change log update